Client/Frontend/Settings/FxAContentViewController.swift

```diff
@@ -128,7 +128,9 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
         helper.delegate = self
         helper.application(app, didReceiveAccountJSON: data)
 
-        LeanplumIntegration.sharedInstance.setUserAttributes(attributes: [UserAttributeKeyName.signedInSync.rawValue : profile.hasAccount()])
```

Comment:
The change was made since profile.hasAccount() is not necessarily true even though the user just signed in (since verification may be required). So now we check in two places to see if the user is truly logged in. Sign out is handled in another file and I've validated that works already.